src/utils/fetchTotalStakeAndAPR.ts

```diff
@@ -0,0 +1,41 @@
+import axios, { AxiosResponse } from 'axios';
+import { BEACONCHAIN_URL, ETH_DEPOSIT_OFFSET } from './envVars';
+
+interface EthstoreData {
+  data: {
+    apr: number;
+    // eslint-disable-next-line camelcase
+    effective_balances_sum_wei: number;
+  };
+}
+export interface FetchTotalStakeAndAPRResponse {
+  statusCode: number;
+  body: {
+    amountEth: number;
+    apr: number;
+    msg?: unknown;
+  };
+}
+export const fetchTotalStakeAndAPR = async (): Promise<FetchTotalStakeAndAPRResponse> => {
+  const { href } = new URL('/api/v1/ethstore/latest', BEACONCHAIN_URL);
+  try {
+    const response: AxiosResponse = await axios.get(href);
+    if (response.status !== 200) throw new Error(response.statusText);
+    const { data }: { data: EthstoreData } = response;
```

Comment:
These types are kind of weird. I think this would simplify it a bit
```suggestion
    // you can pass the data type on the `get` function
    const response = await axios.get<EthstoreData>(href);
    if (response.status !== 200) throw new Error(response.statusText);
    const { data } = response;
```